### ISSUE TITLE
RPD unusable after slipping on banana

### ISSUE THREAD

**Issue by 00aquir** [yogstation13/Yogstation#12613]
### Severity

Inconvenience

### Short Description

Slipping on banana peel made Rapid Pipe Dispenser (RPD) completely unusable.

### Reproduction

- Slipping on banana peel made Rapid Pipe Dispenser (RPD) unusable.
- RPD was in hand with its window active.
- Another RPD needed to be picked up after that, initial RPD could not spawn any kind of tubes or pipes after selection, tried it multiple times during the round.

### Round ID

38006

### BYOND Username

Aquir

**Comment by Xoxeyos** [comment]
I don't believe this is a consistent issue, I tried it out on a private server, and I couldn't get the same results you had.
https://www.youtube.com/watch?v=g3xX-gqxVPs

**Comment by 00aquir** [comment]
It's okay it was just really weird that I literally had to switch to another tool to continue piping.
